bento-compose/build.gradle

```diff
@@ -0,0 +1,52 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+ext.projectName = 'Bento Compose'
+ext.projectDescription = 'A library which provides compatability for Jetpack Compose.'
+
+apply from: "$rootDir/publishing.gradle"
+
+android {
+    compileSdkVersion Versions.COMPILE_SDK
+
+    defaultConfig {
+        minSdkVersion Versions.MIN_SDK
+        targetSdkVersion Versions.TARGET_SDK
+        versionCode 1
+        versionName Publishing.VERSION
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
I don't think it's necessary because we're not really including resources as part of this library. The other modules don't have that set either and it'd be good to keep them all the same.